common/src/main/java/com/sedmelluq/discord/lavaplayer/natives/NativeLibLoader.java

```diff
@@ -85,15 +150,15 @@ public static void load(Class<?> resourceBase, String name) {
   }
 
   private static Path extractLibrary(Class<?> resourceBase, String name) throws IOException {
-    String path = "/natives/" + architecture.systemType + "/" + architecture.libraryPrefix + name + architecture.librarySuffix;
+    String path = "/natives/" + architecture.systemType + "/" + architecture.formatLibraryName(name);
     Path extractedLibrary;
 
     try (InputStream libraryStream = resourceBase.getResourceAsStream(path)) {
       if (libraryStream == null) {
         throw new UnsatisfiedLinkError("Required library at " + path + " was not found");
       }
 
-      File temporaryContainer = new File(System.getProperty("java.io.tmpdir"), "lava-jni-natives/" + libraryDirectory);
+      File temporaryContainer = new File(System.getProperty("java.io.tmpdir"), "lava-jni-natives/" + tempDir);
```

Comment:
Since you're here already, probably makes sense to define this entire directory as a static field. Something like `File extractionDirectory = new File(System.getProperty("java.io.tmpdir"), "lava-jni-natives/" + String.valueOf(System.currentTimeMillis())`.